### ISSUE TITLE
eslint security configuration is not passed to language server properly

### ISSUE THREAD

**Issue by euclio** [creativenull/diagnosticls-configs-nvim#23]
From looking at my language server logs, I see that the eslint security configuration is getting passed as `securities = { "warning", "error" },` when it should be passed as `securities = { "1" = "warning", "2" = "error" }`. I'm not sure where things are going wrong, but it seems like something (neovim, nvim-lspconfig, this plugin?) can't distinguish between a map and an array here.

**Comment by creativenull** [comment]
Can you show how you have it configured? I've tried it on my end and not seeing any issue, able to get both errors and warnings. The `securities` is not ordered in the config but the index is correct. For reference I have tried with the following:

```lua
local dls = require 'diagnosticls-configs'
dls.setup {
  javascript = {
    linter = require 'diagnosticls-configs.linters.eslint'
  },
  typescript = {
    linter = require 'diagnosticls-configs.linters.eslint'
  }
}
```

**Comment by euclio** [comment]
Sure, here's a link to my vimrc: https://github.com/euclio/vimrc/blob/master/vimrc#L491-L505

I get both errors and warnings, but the severity is reported as "1" for all of them according to my LSP logs.

**Comment by euclio** [comment]
Fixed! Thank you.
